### PR TITLE
Contributor Testing Guide: Add section on smooth PR review process

### PR DESCRIPTION
**Pull request by devaips**
This PR is springing from a discussion with @timothysc during the Sig-Arch Conformance Office Hours meeting on 2019-10-08. 

This adds a new section to the test guide describing 4 steps to making a PR review process move smoothly.

### CODE REVIEW

File: contributors/devel/sig-testing/testing.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ passed both unit and integration tests.
 Kubernetes only merges pull requests when unit, integration, and e2e tests are
 passing, so it is often a good idea to make sure the e2e tests work as well.
 
+
+
+
```
Comment (geekygirldawn) [inline contributors/devel/sig-testing/testing.md]: I'm guessing we don't need these extra lines?

File: contributors/devel/sig-testing/testing.md [hunk 2]
```diff
@@ -270,3 +273,19 @@ It is useful as a way to:
...
   - each dashboard is represented as a button
   - some dashboards collect jobs/tests in the domain of a specific SIG (named after and owned by those SIGs), and dashboards to monitor project wide health (owned by SIG-release)
+
+## PR Process
+
+All new PRs for tests should attempt to follow these steps in order to help
+enable a smooth review process:
+
+1. The problem statement should be well defined. (Describe clearly the intended
+purpose of the test, and why it is needed.)
```
Comment (geekygirldawn) [inline contributors/devel/sig-testing/testing.md]: I would maybe consider rewording this to be a bit more concise. Maybe something like: 

"The problem statement should clearly describe the intended purpose of the test and why it is needed."

File: contributors/devel/sig-testing/testing.md [hunk 3]
```diff
@@ -270,3 +273,19 @@ It is useful as a way to:
...
+
+2. Get some agreement on how to design test from relevant SIG.
+
+3. Create the PR.
+
+4. Raise awareness of your PR to respective communities. (eg. via mailing lists,
+Slack channels, Github mentions)
```
Comment (geekygirldawn) [inline contributors/devel/sig-testing/testing.md]: I would maybe change the punctuation to something like:

... to the respective communities (e.g. via mailing lists, Slack channels, Github mentions).

File: contributors/devel/sig-testing/testing.md [hunk 4]
```diff
@@ -270,3 +273,19 @@ It is useful as a way to:
...
+
+## PR Process
+
+All new PRs for tests should attempt to follow these steps in order to help
+enable a smooth review process:
+
+1. The problem statement should be well defined. (Describe clearly the intended
+purpose of the test, and why it is needed.)
+
+2. Get some agreement on how to design test from relevant SIG.
```
Comment (geekygirldawn) [inline contributors/devel/sig-testing/testing.md]: I would maybe reword this to:

"Get some agreement on how to design your test from the relevant SIG."

File: contributors/devel/sig-testing/testing.md (hunk 1 above)
Comment (devaips) [reply contributors/devel/sig-testing/testing.md]: Correct! Fixed in https://github.com/kubernetes/community/pull/4170/commits/af108e4f6e1c819992843e4c8dde06ffb52d5e70

File: contributors/devel/sig-testing/testing.md (hunk 2 above)
Comment (devaips) [reply contributors/devel/sig-testing/testing.md]: Changed in af108e4

File: contributors/devel/sig-testing/testing.md (hunk 3 above)
Comment (devaips) [reply contributors/devel/sig-testing/testing.md]: Changed in af108e4

File: contributors/devel/sig-testing/testing.md (hunk 4 above)
Comment (devaips) [reply contributors/devel/sig-testing/testing.md]: Changed in af108e4.

Thanks for all the recommendations!
